acctests/README.md

```diff
@@ -72,19 +54,14 @@ $ terraform apply
 ```
 
 ## Run Terraform Phase 2
-Phase 1 should have created a file with more environment variables called `devrc`. You should source that file, and then re-source wherever you saved your previous environment variables (so the required secrets get picked up).
+A file with several environment variables called `devrc` should have been created within `acctests/equinix`. You should source that file and then switch to `acctests/vsphere`.
 ```
 $ source devrc
-$ source ~/.zshenv # or wherever you set them
-$ cp main.tf.phase2 main.tf
-$ terraform apply
-```
-There is an issue where a network is unavailable right away and some test VMs will fail to create, running a second apply should fix this.
-```
+$ cd ../vsphere
+$ terraform init
 $ terraform apply
 ```
-Now all the infrastructure should be created, if you planned on running the cluster tests with vSAN, make sure to enable vSAN in VMKernal Adapters on the management network for ESXI hosts 3 and 4. This is must be done through the web console. You should have most of the necessary environment variables for acceptances tests by sourcing
+Now all the infrastructure should be created, if you planned on running the cluster tests with vSAN, make sure to enable vSAN in VMKernal Adapters on the management network for ESXI hosts 3 and 4. This is must be done through the web console. Another `devrc` file should have been created within the `acctests/vsphere` folder, source that now and you should have all the necessary environment variables for running tests.
```

Comment:
```suggestion
Now all the infrastructure should be created, if you planned on running the cluster tests with vSAN, make sure to enable vSAN on the VMkernal adapters on the management network for ESXI hosts 3 and 4. This can be done through the vSphere client or PowerCLI.  An additional `devrc` file should have been created in the `acctests/vsphere` path. Source this file and you should have all the necessary environment variables for running tests.
```